logging/config/cluster-logging-external.adoc

```diff
@@ -5,9 +5,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can send Elasticsearch logs to external devices, such as an externally-hosted Elasticsearch instance
+{product-title} cluster logging allows send Elasticsearch logs to external devices, such as an externally-hosted Elasticsearch instance
```

Comment:
I think `Elasticsearch logs` is not correct, maybe `container and node logs ` or just `logs`